container-guide/concepts/containers-bci-app.xml

```diff
@@ -20,7 +20,7 @@
   </info>
   <para>
     Application &bcia;s are &bcia;-Base container images that include specific
```

Comment:
The output is _Application SLE BCIs are SLE BCI-Base container images that include specific applications_ and I wonder whether we should reword this to _Application SLE BCIs are SLE Base Container Images that include specific applications_ ?